### PR TITLE
Modify Shutdown Dialog Modal

### PR DESCRIPTION
**Pull request by djclueless**
Shutdown Dialog converted to Modal

### CODE REVIEW

File: app/static/css/style.css [hunk 1]
```diff
@@ -14,10 +14,27 @@ body {
...
+    bottom: 0;
+    background-color: rgba(0,0,0,0.8);
+    z-index: 2;
+}
+
 * {
   font-family: Lora;
 }
 
+*:focus {
```
Comment (mtlynch) [inline app/static/css/style.css:34]: Looks like this accidentally got pulled in from #202. Could we rebase onto master to get rid of this?

File: app/static/js/app.js [hunk 2]
```diff
@@ -346,6 +347,12 @@ document
 document.getElementById("cancel-shutdown").addEventListener("click", () => {
   hideElementById("shutdown-confirmation-panel");
 });
+document.getElementById("shutdown-confirmation-panel").addEventListener("click", (e) => {
```
Comment (mtlynch) [inline app/static/js/app.js:350]: Can we rename `e` to `evt` for consistency with other functions that use this convention?

File: app/static/js/app.js [hunk 3]
```diff
@@ -346,6 +347,12 @@ document
...
+  e = window.event || e;
+  if(e.target.className === "overlay") {
```
Comment (mtlynch) [inline app/static/js/app.js:352]: The spacing here is causing the build check to fail. Could you run the formatter so it matches the JS/CSS whitespace conventions?

```bash
npm install npm install prettier@2.0.5
./node_modules/.bin/prettier --write app/static
```

**Review by mtlynch — CHANGES_REQUESTED** [review]
Nice addition! Just a few small tweaks.

**Comment by djclueless** [comment]
I give up submitting a new request, I am stuck in a rebase loop. Sending a different PR.

**Pull request closed without merging**